string collsion_obj_name
string mesh_path
---
bool success